gmso/tests/test_forcefield.py

```diff
@@ -274,26 +276,34 @@ def test_ff_periodic_dihedrals_from_alphanumeric_symbols(self):
             assert len(
                 ff.dihedral_types["opls_140~*~*~opls_140"].parameters["c0"]
             )
-        assert len(ff.dihedral_types["NH2~CT1~C~O"].parameters["delta"]) == 1
+        assert ff.dihedral_types["NH2~CT1~C~O"].parameters[
+            "delta"
+        ] == u.unyt_quantity(0.0, "degree")
 
     def test_ff_from_etree(self):
-        ff_etree = lxml.etree.parse(get_path("opls_charmm_buck.xml"))
-        ff = ForceField(ff_etree)
+        # TODO: load using backend forcefield-utilities from etree
+        ff_etree = lxml.etree.parse(get_path("ethylene.xml"))
+        ff = ForceField(ff_etree, backend="gmso")
         assert ff
 
     def test_ff_from_etree_iterable(self):
+        # TODO: load using backend forcefield-utilities from etree
         ff_etrees = [
-            lxml.etree.parse(get_path("opls_charmm_buck.xml")),
-            lxml.etree.parse(get_path("trimmed_charmm.xml")),
+            lxml.etree.parse(get_path("carbon.xml")),
+            lxml.etree.parse(get_path("ethylene.xml")),
```

Comment:
Fixed in commit 93d813f82a66dc81a825042b283a26b3b4890c96. I had changed them to see if the tests passed for these xmls because they didn't for the old versions of the charmm and buck forcefields, but then once I got the okay to manually change their formatting to match the updated formats, they pass the tests now fine. So I changed it back to test the charmm and buck ffs again.